docs/sources/articles/dockerfile_best-practices.md

```diff
@@ -40,6 +40,8 @@ context and final image. For example, unless`.git` is needed by your build
 process or scripts, you should add it to `.dockerignore`, which can save many
 megabytes worth of upload time.
 
+The `.dockerignore` file also supports exclusion patterns (as per `.gitignore` files).
```

Comment:
Add to the paragraph above.  Also, reduce complexity in the original paragraph (not your fault).

--> https://gist.github.com/moxiegirl/88f830e38843be32a1df

For faster uploading and efficiency during `docker build`, you should use
a `.dockerignore` file. This file excludes files or directories from the build context and final image. For example, unless`.git` is needed by your build
process or scripts, you should add it to `.dockerignore`. This can save many
megabytes worth of upload time. The `.dockerignore` file also supports the same exclusion patterns as a `.gitignore` file.